components/Navbar.vue

```diff
@@ -145,173 +143,124 @@ import ExploreDropdown from '~/components/navbar/ExploreDropdown.vue'
 import CreateDropdown from '~/components/navbar/CreateDropdown.vue'
 import KodaBetaDark from '@/assets/Koda_Beta_dark.svg'
 import KodaBeta from '@/assets/Koda_Beta.svg'
-import PrefixMixin from '@/utils/mixins/prefixMixin'
 import ColorModeButton from '~/components/common/ColorModeButton.vue'
 import MobileLanguageOption from '~/components/navbar/MobileLanguageOption.vue'
 import { createVisible } from '@/utils/config/permision.config'
-import AuthMixin from '@/utils/mixins/authMixin'
-import ExperimentMixin from '@/utils/mixins/experimentMixin'
 import ChainSelectDropdown from '~/components/navbar/ChainSelectDropdown.vue'
 import StatsDropdown from '~/components/navbar/StatsDropdown.vue'
 import MobileNavbarProfile from '~/components/navbar/MobileNavbarProfile.vue'
 import ConnectWalletButton from '~/components/shared/ConnectWalletButton.vue'
-import { getKusamaAssetId } from '~/utils/api/bsx/query'
-import { NeoButton } from '@kodadot1/brick'
 import { ConnectWalletModalConfig } from '@/components/common/ConnectWallet/useConnectWallet'
+import { BModalConfig } from 'buefy/types/components'
+import Vue from 'vue'
 
-@Component({
-  components: {
-    NeoButton,
-    Search,
-    BasicImage,
-    MobileExpandableSection,
-    ProfileDropdown,
-    ExploreDropdown,
-    CreateDropdown,
-    ChainSelectDropdown,
-    StatsDropdown,
-    MobileNavbarProfile,
-    NavbarExploreOptions,
-    ConnectWalletButton,
-    ColorModeButton,
-    MobileLanguageOption,
-  },
-})
-export default class NavbarMenu extends mixins(
-  PrefixMixin,
-  AuthMixin,
-  ExperimentMixin
-) {
-  public showTopNavbar = true
-  public openMobileSearchBar = false
-  private fixedTitleNavAppearDistance = 85
-  private lastScrollPosition = 0
-  private artistName = ''
-  public isBurgerMenuOpened = false
-  private isMobile = window.innerWidth < 1024
+const { $store, $colorMode, $buefy, $nextTick } = useNuxtApp()
+const root = ref<Vue<Record<string, string>>>()
+const showTopNavbar = ref(true)
+const openMobileSearchBar = ref(false)
+const fixedTitleNavAppearDistance = ref(85)
+const lastScrollPosition = ref(0)
+const isBurgerMenuOpened = ref(false)
+const isMobile = ref(window.innerWidth < 1024)
+const { urlPrefix } = usePrefix()
 
-  @Ref('mobilSearchRef') readonly mobilSearchRef
+const mobilSearchRef = ref<{ focusInput: () => void } | null>(null)
 
-  get account() {
-    return this.$store.getters.getAuthAddress
-  }
+const route = useRoute()
 
-  get chain(): string {
-    return this.urlPrefix
-  }
+const account = computed(() => $store.getters.getAuthAddress)
 
-  get tokens() {
-    return ['', getKusamaAssetId(this.urlPrefix)]
-  }
+const isCreateVisible = computed(() => createVisible(urlPrefix.value))
+const isLandingPage = computed(() => route.name === 'index')
 
-  get isCreateVisible(): boolean {
-    return createVisible(this.urlPrefix)
-  }
+const isDarkMode = computed(
```

Comment:
Isnt there composable from vueuse? Cc @roiLeo

Reply:
```ts
const colorMode = useColorMode()
$consola.log(colorMode.preference)
```

`$colorMode` from `useNuxtApp` working tambien

Reply:
context: You will get `system` instead of `dark/light` value, when you visit the preview env the first time and have not set any color mode. So, I introduced this method to solve the incorrect color mode issue.  
<img width="242" alt="image" src="https://user-images.githubusercontent.com/31397967/220245981-f60f51c7-c73c-41ba-a121-529d2743234d.png">
   
IMO, we should make this solution a global composable method and use it everywhere to replace `useColorMode`. That could be a follow-up issue.